module.json

```diff
@@ -7,7 +7,8 @@
   "homepage": "https://github.com/ARMmbed/mbed-client",
   "license": "Apache-2.0",
   "dependencies": {
-    "mbed-client-c": "^2.0.0"
+    "mbed-client-c": "^2.0.0",
+    "mbed-trace": "ARMmbed/mbed-trace"
```

Comment:
Why still do it like mbed-trace": "ARMmbed/mbed-trace#0.2.1 and not like
mbed-trace": "0.2.1" ?